### ISSUE TITLE
Left/Right channels do not seem to work for Wav File load

### ISSUE THREAD

**Issue by ac2cz** [ac2cz/FoxTelem#483]
Ubuntu Linux 18.04 using foxtelem 1.06r

"I split the stereo that had the good audio (with frames) and exported it as two files; left and right, where only one side of the stereo file had audio.  
The left side works, but the right side doesn't.  No matter which you choose in settings.  Checking and un-checking the "Use left stereo channel" has no effect.  
So with the audio file that only has right channel, nothing... ever... checked or unchecked."

This seems to work on Windows, but should be rechecked for 1.06r.  There seems to be a difference in Java audio for Linux that means this does not work.

**Comment by ac2cz** [comment]
In fact this is a bug with the Wav file load directly and not tied to Linux.  The Wav loader does not look at the left/right flag.  The SoundCard loader does.  Need to make sure that stereo files are processed correctly for wav file load.

**Comment by ac2cz** [comment]
Fixed in 1.07r but needs testing.  I don't have a suitable file.

**Comment by lisa50469** [comment]
Just tested 1.07r version, using a recorded WAV file I had saved.  The recorded WAV was from my V71A, with 9600 data on the left side, and 1200 on the right.  So when I ran it through foxtelem 'r' left side I got 56 frames.  When I ran it again after deleting payloads, I go no frames when using the right side.  Looks like this is fixed. 
I'll do more testing with live audio when I'm setup again... rainy here for the next few days.
